tests/core/activity/registry_test.py

```diff
@@ -222,3 +222,40 @@ async def test_slash_command(activity_registry, message_sent):
     assert slash_activity._name == command_name
     assert slash_activity._requires_mention_bot == mention_bot
     assert slash_activity._callback == listener
+
+
+@pytest.mark.asyncio
+async def test_slash_same_command_name_and_mention(activity_registry, message_sent):
+    listener = AsyncMock()
+    mention_bot = True
+    command_name = "/command"
+
+    activity_registry.slash(command_name, mention_bot)(listener)
```

Comment:
Please consider having two different listener instances to check we do not blindly make a reference equality check

Reply:
you are right, I missed that. Done in https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/pull/195/commits/cd17c57c6512282cc8ad25df885042bc2a287681